### PR TITLE
fix: Update example paths.

### LINKED ISSUE #726: Code examples in app show wrong host and path
They should show `https://api.unkey.dev/v1/keys.verifyKey` or `http://localhost:8787/v1/keys.verifyKey`

### PR DESCRIPTION
**Pull request by perkinsjr**
## What does this PR do?

closes #726 

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How should this be tested?

- Test A
- Test B

### Required

- [X] Filled out the "How to test" section in this PR
- [X] Read [Contributing Guide](./CONTRIBUTING.md)
- [X] Self-reviewed my own code
- [X] Commented on my code in hard-to-understand areas
- [X] Ran `pnpm build`
- [X] Ran `pnpm fmt`
- [X] Checked for warnings, there are none
- [X] Removed all `console.logs`
- [X] Merged the latest changes from main onto my branch with `git pull origin main`
- [X] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the Unkey Docs if changes were necessary

### CODE REVIEW

File: apps/web/app/(authenticated)/(app)/new/keys.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export const Keys: React.FC<Props> = ({ apiId }) => {
 
   const createKeySnippet = `curl -XPOST '${
     process.env.NEXT_PUBLIC_UNKEY_API_URL ?? "https://api.unkey.dev"
-  }/v1/keys' \\
+  }/v1/keys.verifyKey' \\
```
Comment (chronark) [inline apps/web/app/(authenticated)/(app)/new/keys.tsx:81]: @perkinsjr holdup
this is `/v1/keys.createKey`

File: apps/web/app/(authenticated)/(app)/app/settings/root-keys/create-root-key-button.tsx [hunk 2]
```diff
@@ -42,7 +42,7 @@ export const CreateRootKeyButton: React.FC<Props> = ({ apiId }) => {
     },
   });
 
-  const snippet = `curl -XPOST '${process.env.NEXT_PUBLIC_UNKEY_API_URL ?? "https://api.unkey.dev"}/v1/keys' \\
+  const snippet = `curl -XPOST '${process.env.NEXT_PUBLIC_UNKEY_API_URL ?? "https://api.unkey.dev"}/v1/keys.verifyKey' \\
```
Comment (chronark) [inline apps/web/app/(authenticated)/(app)/app/settings/root-keys/create-root-key-button.tsx:45]:
```suggestion
  const snippet = `curl -XPOST '${process.env.NEXT_PUBLIC_UNKEY_API_URL ?? "https://api.unkey.dev"}/v1/keys.createKey' \\
```

File: apps/web/app/(authenticated)/(app)/new/keys.tsx (hunk 1 above)
Comment (chronark) [inline apps/web/app/(authenticated)/(app)/new/keys.tsx:81]:
```suggestion
  }/v1/keys.createKey' \\
```

**Review by chronark — CHANGES_REQUESTED** [review]

File: apps/web/app/(authenticated)/(app)/new/keys.tsx (hunk 1 above)
Comment (perkinsjr) [reply apps/web/app/(authenticated)/(app)/new/keys.tsx]: Copy pasta.

**Review by chronark — APPROVED** [review]

**Pull request merged**